packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -56,20 +56,36 @@ export default class TelecomPackMigrationOffersCtrl {
 
   updateOfferDisplayedPrice(value, offer, optionName) {
     let totalOfferPrice = offer.price.value;
+    let optionComfort = 0;
 
     angular.forEach(offer.options, (option) => {
-      if (option.name === 'gtr_ovh' && option.selected) {
-        totalOfferPrice += option.optionalPrice.value;
+      if (
```

Comment:
The `forEach` loop makes two different things. Could the count, and the is option selected logics be done separately?

Reply:
done I made a static function for selected logic